tests/common/DidServiceEndpoint.ts

```diff
@@ -7,14 +7,11 @@ export default class DidServiceEndpoint {
   /**
    * Test if a MongoDB service is running at the specified url.
    */
-  public static createHubServiceEndpoint (instances: string[]): DidServiceEndpointModel {
```

Comment:
Can we remove this class altogether or maybe tweak the method to be more general purpose because:
1. "Hub" is no longer a used term.
1. This method has become quite pointless, if it's only not hardcoding ID.
1. The description became incorrect at some point.